### ISSUE TITLE
Virtual methods are wrongly marked in docs and for godot's gdscript editor

### ISSUE THREAD

**Issue by Yarwin** [godot-rust/gdext#917]
Given some virtual method

```rust
    #[func(virtual)]
    pub fn my_virtual_method(&self) {
        );
    }
```

It will display in Godot's internal docs as
![image](https://github.com/user-attachments/assets/9776cdf8-80b6-434c-8193-55a3c057468c)

and in editor as
![image](https://github.com/user-attachments/assets/cdc1a416-2eff-42fa-b7cf-3882773a7776)
![image](https://github.com/user-attachments/assets/7d2cbc49-aeb9-410e-a03f-fd68f03e874a)

Meanwhile the real function call requires end-user to annotate given method name with `_`
for example, following code, called in rust by something along the lines of `my_rust_class.bind().my_virtual_method()`
```gdscript
extends MyRustClass
class_name GdscriptClass

func my_virtual_method() -> void:
	print("I am never called!")

func _my_virtual_method() -> void:
	print("hello from gdscript!")
```

Will output "hello from gdscript".

Excepted behavior: correct names of given methods in docs and editor's autocomplete. Not sure about get_method_list tho

**Comment by Yarwin** [comment]
Note – this behavior is [properly documented](https://docs.rs/godot/0.1.3/godot/prelude/attr.godot_api.html#virtual-methods). It is just very confusing for end users :grimacing:

**Comment by Bromeon** [comment]
Thanks! Thats a mistake on our side, I fixed it in #918. Could you maybe check if it works for you too?

---

A bit of background on `method` vs `_method`: there are some [known limitations](https://godot-rust.github.io/book/register/virtual-functions.html#limitations), particularly point 1. From outside Rust, it's not possible to call the virtual function polymorphically -- only the overridden GDScript method.

I was considering to eventually add something like `#[func(virtual_pub)]` (better naming), which would register the polymorphic version as `my_virtual_method`. It might even be possible to do that for all methods, but maybe registering an extra non-virtual method isn't necessary in most cases.
